### ISSUE TITLE
Linking particles to molecular entities

### ISSUE THREAD

**Issue by vedina** [enanomapper/ontologies#49]
Is there a link in the ENM ontology, allowing to represent the `zinc oxide nanoparticle` and the `zinc oxide` [CHEBI_36560](http://bioportal.bioontology.org/ontologies/CHEBI?p=classes&conceptid=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FCHEBI_36560) are related?

I'm looking for automatic means to retrieve this linking information and support user friendly search, as people will be usually searching for `ZnO` and not necessary for `ZnO nanoparticle`, while expecting the nanoparticles to show up.

**Comment by egonw** [comment]
Yeah, the NPO shows how this can be done. See this patch: https://github.com/sobolevnrm/npo/commit/b24c95614e3a3928c70b6e90a2445a2dc77cd24d

Particularly, check lines 14472-14482 ...

So, the link is not a single predicate...

**Comment by fehrhart** [comment]
close?

**Comment by vedina** [comment]
an example would be nice
